diff --git a/node_modules/react-magnifier/dist/Magnifier.d.ts b/node_modules/react-magnifier/dist/Magnifier.d.ts
index 5978345..7e8adea 100644
--- a/node_modules/react-magnifier/dist/Magnifier.d.ts
+++ b/node_modules/react-magnifier/dist/Magnifier.d.ts
@@ -25,7 +25,7 @@ interface State {
     relX: number;
     relY: number;
 }
-export default class Magnifier extends PureComponent<Props, State> {
+ class Magnifier extends PureComponent<Props, State> {
     static defaultProps: {
         width: string;
         height: string;
@@ -58,5 +58,6 @@ export default class Magnifier extends PureComponent<Props, State> {
     calcImgBounds: () => void;
     render: () => React.ReactElement<any, string | ((props: any) => React.ReactElement<any, string | any | (new (props: any) => React.Component<any, any, any>)> | null) | (new (props: any) => React.Component<any, any, any>)>;
 }
+ export default  React.FunctionComponent<Magnifier>
 export {};
 //# sourceMappingURL=Magnifier.d.ts.map
\ No newline at end of file
